src/event/trigger.js

```diff
@@ -150,26 +150,20 @@ jQuery.extend( jQuery.event, {
 	},
 
 	// Piggyback on a donor event to simulate a different one
+	// Used only for `focus(in | out)` events
 	simulate: function( type, elem, event ) {
 		var e = jQuery.extend(
 			new jQuery.Event(),
 			event,
 			{
 				type: type,
-				isSimulated: true
-
-				// Previously, `originalEvent: {}` was set here, so stopPropagation call
-				// would not be triggered on donor event, since in our own
-				// jQuery.event.stopPropagation function we had a check for existence of
-				// originalEvent.stopPropagation method, so, consequently it would be a noop.
-				//
-				// But now, this "simulate" function is used only for events
-				// for which stopPropagation() is noop, so there is no need for that anymore.
-				//
-				// For the compat branch though, guard for "click" and "submit"
-				// events is still used, but was moved to jQuery.event.stopPropagation function
-				// because `originalEvent` should point to the original event for the constancy
-				// with other events and for more focused logic
+				isSimulated: true,
+
+				// Support: Firefox <= 46
+				// In Firefox, called `stop(Immediate)Propagation` methods, in
+				// capturing phase, prevents receiving focus,
+				// so we stop only jQuery propagation not the native one instead
```

Comment:
Methods (plural) prevent focus, not prevents, there are too many commas in the sentence, the action of calling the method prevents receiving focus and not the methods themselves, `only` sounds weird with `instead`, etc. You'll see that if you compare your comment with mine.

I've seen now that I made a mistake in my version as well: after changing `methods` to `calling` as the subject it's `prevents` again. Fixed.